Frameworks/UIKit/UIButton.mm

```diff
@@ -289,6 +284,7 @@ - (void)layoutSubviews {
     CGRect imageFrame = [self imageRectForContentRect:contentFrame];
     self.titleLabel.frame = textFrame;
     self.imageView.frame = imageFrame;
+    UIImageSetLayerContents([self layer], self.currentBackgroundImage);
```

Comment:
contents behind pre-canned subviews do make sense, but this should still get a comment

---
In reply to: [87647058](https://github.com/Microsoft/WinObjC/pull/1361#discussion_r87647058) [](ancestors = 87647058)